StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -116,6 +121,10 @@ public void HandleMethodDeclaration(SyntaxNodeAnalysisContext context)
                 }
 
                 MethodDeclarationSyntax declaration = (MethodDeclarationSyntax)context.Node;
+                if (!(declaration.Parent is BaseTypeDeclarationSyntax))
```

Comment:
It would just return the `declaredAccessibility` value if the parent is not a type.